### PR TITLE
Fixes circular dependency that prevents a unified Deposit class

### PR DESCRIPTION
**Pull request by dangerousfood**
I am attempting to unify the two Deposit classes into a single class. To do that the dependency tree needs to be fixed.

### CODE REVIEW

**Review by cemozerr — APPROVED** [review]
lgtm

File: data/build.gradle [hunk 1]
```diff
@@ -9,11 +9,11 @@ jar {
 dependencies {
   implementation project(':util')
   implementation project(':ethereum:datastructures')
+  implementation 'com.fasterxml.jackson.core:jackson-databind'
```
Comment (jrhea) [inline data/build.gradle]: curious why you bothered moving this.  its better to keep 3rd party dependencies grouped.  actually i dont give af, but people judge

File: ethereum/datastructures/build.gradle [hunk 2]
```diff
@@ -8,6 +8,8 @@ jar {
 
 dependencies {
   implementation project(':util')
+  implementation project(':pow')
+  compile group: 'com.googlecode.json-simple', name: 'json-simple', version: '1.1'
```
Comment (jrhea) [inline ethereum/datastructures/build.gradle]: mind removing this?

**Review by jrhea — APPROVED** [review]
just a couple of nitpicks to fix.  otherwise, lgtm

File: data/build.gradle (hunk 1 above)
Comment (dangerousfood) [reply data/build.gradle]: Totally by accident.

**Pull request merged**